pyamg/aggregation/adaptive.py

```diff
@@ -113,7 +113,7 @@ def unpack_arg(v):
 def adaptive_sa_solver(A, initial_candidates=None, symmetry='hermitian',
                        pdef=True, num_candidates=1, candidate_iters=5,
                        improvement_iters=0, epsilon=0.1,
-                       max_levels=10, max_coarse=100, aggregate='standard',
```

Comment:
Hmm... The 200x200 solve is really only fast because it calls LAPACK. From a FLOP perspective, I think that is still C*8,000,000, for some non-trivial C. I guess what I'm getting at is if we really optimized everything else in the code, would we actually use a direct solve on that large of a problem?? It seems to me it could bump CC pretty hard. What does Hypre do on the direct solve?